src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ForkJoinPool;
+
+public class Main {
+    private static final int SIZE = 1000000;
+    private static final int NUMOFTHREADS =
```

Comment:
```suggestion
    private static final int NUM_OF_THREADS =
```
don't forget about constant naming convention